components/table/style/radius.ts

```diff
@@ -33,6 +33,7 @@ const genRadiusStyle: GenerateStyle<TableToken, CSSObject> = (token) => {
         '&-container': {
           borderStartStartRadius: tableRadius,
           borderStartEndRadius: tableRadius,
+          overflowX: 'hidden',
```

Comment:
This may break user who use absolute to do some hack position which will be cut.